it/uk/gov/hmrc/eusubsidycompliancefrontend/connector/EscConnectorSpec.scala

```diff
@@ -0,0 +1,483 @@
+/*
```

Comment:
Changed to use wiremock. Also actually tests the right implicits are passed as it affects transformation of 404's etc.